gcp/modules/dcos-tested-gcp-oses/main_variables.tf

```diff
@@ -0,0 +1,6 @@
+variable "provider" {
+ default = "gcp"
+}
+
+variable "os" {}
+variable "region" {}
```

Comment:
is there a reason `os` and `region` are empty by default?

Reply:
the actual values of `os` and `region` are populated and required by the top level root module github.com/dcos/terraform-dcos/aws github.com/dcos/terraform-dcos/azure, etc. 

By listing the values of os and region would be inapplicable.